Deb package creation errors

Добрый день, инструкция по сборке deb пакетов немного устарела, вместо `make deb` сейчас используется `make make_deb`. Однако `make make_deb` тоже не работает.
```
$ make make_deb
rm -fr .build* sviewgl sviewgl .compile_commands \
	sviewgl-3.2.2* sviewgl_3.2.2* \
	*.log *.tasks *.sh *.xz *.list *.deb \
	cppcheck-output strace_out cov-int
install -d sviewgl-3.2.2
cp -R cmake src res dist/debian INSTALL README.md CMakeLists.txt Makefile sviewgl.desktop res/Icon-1024.png dist/fedora/* dist/gentoo/* sviewgl-3.2.2
mv sviewgl-3.2.2/simpleviewer-gl-_VERSION_.ebuild sviewgl-3.2.2/simpleviewer-gl-3.2.2.ebuild
sed "s/_VERSION_/3.2.2/" -i sviewgl-3.2.2/sviewgl.spec
sed "s/_VERSION_/3.2.2/" -i sviewgl-3.2.2/debian/changelog
tar -zf sviewgl-3.2.2.tar.gz -c sviewgl-3.2.2
mv sviewgl-3.2.2.tar.gz sviewgl_3.2.2.orig.tar.gz
cd sviewgl-3.2.2 ; PREFIX=/usr dpkg-buildpackage -F -tc
dpkg-buildpackage: инфо: пакет исходного кода sviewgl
dpkg-buildpackage: инфо: версия исходного кода 3.2.2-1
dpkg-buildpackage: инфо: дистрибутив исходного кода unstable
dpkg-buildpackage: инфо: исходный код изменён Andrey Ugolnik <andrey@ugolnik.info>
dpkg-buildpackage: инфо: архитектура узла amd64
 dpkg-source --before-build .
 debian/rules clean
make[1]: вход в каталог «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
dh clean 
dh: warning: Compatibility levels before 10 are deprecated (level 9 in use)
   dh_auto_clean
dh_auto_clean: warning: Compatibility levels before 10 are deprecated (level 9 in use)
	make -j1 clean
make[2]: вход в каталог «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
rm -fr .build* sviewgl sviewgl .compile_commands \
	sviewgl-3.2.2* sviewgl_3.2.2* \
	*.log *.tasks *.sh *.xz *.list *.deb \
	cppcheck-output strace_out cov-int
make[2]: выход из каталога «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
   dh_clean
dh_clean: warning: Compatibility levels before 10 are deprecated (level 9 in use)
make[1]: выход из каталога «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
 dpkg-source -b .
dpkg-source: инфо: используется формат исходника «3.0 (quilt)»
dpkg-source: инфо: сборка sviewgl с использованием существующего ./sviewgl_3.2.2.orig.tar.gz
dpkg-source: инфо: сборка sviewgl в sviewgl_3.2.2-1.debian.tar.xz
dpkg-source: инфо: сборка sviewgl в sviewgl_3.2.2-1.dsc
 debian/rules binary
make[1]: вход в каталог «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
dh binary 
dh: warning: Compatibility levels before 10 are deprecated (level 9 in use)
   dh_update_autotools_config
   dh_auto_configure
dh_auto_configure: warning: Compatibility levels before 10 are deprecated (level 9 in use)
   dh_auto_build
dh_auto_build: warning: Compatibility levels before 10 are deprecated (level 9 in use)
	make -j1
make[2]: вход в каталог «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
Usage:
    make <release | debug>    - make release or debug application
    make install              - install application
    make <cppcheck>           - do static code verification
    make <make_tar>           - make tar.gz source package
    make <make_deb>           - make DEB package
    make <make_rpm>           - make RPM package
    make <clean>              - cleanup directory
make[2]: выход из каталога «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
   dh_auto_test
dh_auto_test: warning: Compatibility levels before 10 are deprecated (level 9 in use)
   dh_prep
   dh_auto_install --destdir=debian/sviewgl/
dh_auto_install: warning: Compatibility levels before 10 are deprecated (level 9 in use)
	make -j1 install DESTDIR=/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2/debian/sviewgl AM_UPDATE_INFO_DIR=no
make[2]: вход в каталог «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
install -m 755 -d /home/abravo/build/simple-viewer-gl/sviewgl-3.2.2/debian/sviewgl/usr/bin
install -m 755 sviewgl /home/abravo/build/simple-viewer-gl/sviewgl-3.2.2/debian/sviewgl/usr/bin
install: не удалось выполнить stat для 'sviewgl': Нет такого файла или каталога
make[2]: *** [Makefile:88: install] Ошибка 1
make[2]: выход из каталога «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
dh_auto_install: error: make -j1 install DESTDIR=/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2/debian/sviewgl AM_UPDATE_INFO_DIR=no returned exit code 2
make[1]: *** [debian/rules:18: binary] Ошибка 255
make[1]: выход из каталога «/home/abravo/build/simple-viewer-gl/sviewgl-3.2.2»
dpkg-buildpackage: ошибка: debian/rules binary subprocess returned exit status 2
make: *** [Makefile:100: make_deb] Ошибка 2
```

--- a/Makefile
+++ b/Makefile
@@ -19,14 +19,16 @@ ifeq ($(UNAME), Darwin)
 	OUT_NAME="Simple Viewer GL.app"
 endif
 
-all:
+all:    release
+
+help:
 	@echo "Usage:"
 	@echo "    make <release | debug>    - make release or debug application"
 	@echo "    make install              - install application"
 	@echo "    make <cppcheck>           - do static code verification"
-	@echo "    make <make_tar>           - make tar.gz source package"
-	@echo "    make <make_deb>           - make DEB package"
-	@echo "    make <make_rpm>           - make RPM package"
+	@echo "    make <dist>               - make tar.gz source package"
+	@echo "    make <deb>                - make DEB package"
+	@echo "    make <rpm>                - make RPM package"
 	@echo "    make <clean>              - cleanup directory"
 
 release:
@@ -87,7 +89,7 @@ install:
 	install -m 755 -d $(DESTDIR)$(PREFIX)/bin
 	install -m 755 $(BUNDLE_NAME) $(DESTDIR)$(PREFIX)/bin
 
-make_tar: clean
+dist:   clean
 	install -d $(BUNDLE_NAME)-$(VERSION)
 	cp -R cmake src res dist/debian INSTALL README.md CMakeLists.txt Makefile sviewgl.desktop res/Icon-1024.png dist/fedora/* dist/gentoo/* $(BUNDLE_NAME)-$(VERSION)
 	mv $(BUNDLE_NAME)-$(VERSION)/simpleviewer-gl-_VERSION_.ebuild $(BUNDLE_NAME)-$(VERSION)/simpleviewer-gl-$(VERSION).ebuild
@@ -95,9 +97,9 @@ make_tar: clean
 	sed "s/_VERSION_/$(VERSION)/" -i $(BUNDLE_NAME)-$(VERSION)/debian/changelog
 	tar -zf $(BUNDLE_NAME)-$(VERSION).tar.gz -c $(BUNDLE_NAME)-$(VERSION)
 
-make_deb: make_tar
+deb:    dist
 	mv $(BUNDLE_NAME)-$(VERSION).tar.gz $(BUNDLE_NAME)_$(VERSION).orig.tar.gz
 	cd $(BUNDLE_NAME)-$(VERSION) ; PREFIX=/usr dpkg-buildpackage -F -tc
 
-make_rpm: make_tar
+rpm:    dist
 	rpmbuild -ta $(BUNDLE_NAME)-$(VERSION).tar.gz

--- a/Makefile
+++ b/Makefile
@@ -91,7 +91,7 @@ install:
 
 dist:   clean
 	install -d $(BUNDLE_NAME)-$(VERSION)
-	cp -R cmake src res dist/debian INSTALL README.md CMakeLists.txt Makefile sviewgl.desktop res/Icon-1024.png dist/fedora/* dist/gentoo/* $(BUNDLE_NAME)-$(VERSION)
+	cp -R cmake src res third-party dist/debian INSTALL README.md CMakeLists.txt Makefile sviewgl.desktop res/Icon-1024.png dist/fedora/* dist/gentoo/* $(BUNDLE_NAME)-$(VERSION)
 	mv $(BUNDLE_NAME)-$(VERSION)/simpleviewer-gl-_VERSION_.ebuild $(BUNDLE_NAME)-$(VERSION)/simpleviewer-gl-$(VERSION).ebuild
 	sed "s/_VERSION_/$(VERSION)/" -i $(BUNDLE_NAME)-$(VERSION)/$(BUNDLE_NAME).spec
 	sed "s/_VERSION_/$(VERSION)/" -i $(BUNDLE_NAME)-$(VERSION)/debian/changelog

